src/components/Dashboard/AccountBanner.js

```diff
@@ -176,7 +174,7 @@ const BannerWithProbability = ({ activeBalance }) => {
   // Calculate probability (since the total active balance is asynconous
   // it can happen that it has not been updated yet when the juror active balance has)
   const draftingProbability = Math.min(1, totalPercentage / 100)
-  const probabilityText = getProbabilityText(draftingProbability)
+  const chances = totalPercentage > 0 ? Math.floor(100 / totalPercentage) : 0
```

Comment:
We most likely want a better number than falling back to `0`. This is going to be wildly confusing :)

<img width="849" alt="Screen Shot 2020-02-13 at 12 32 18 AM" src="https://user-images.githubusercontent.com/4166642/74387241-4d294580-4df8-11ea-8d62-232c571546c7.png">